src/ui/server/helpers/mappings/map-currencies/is-supported-currency/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import isSupportedCurrency from '.';
+import { GBP, AED } from '../../../../constants';
```

Comment:
Same as above, could we use `NON_STANDARD_CURRENCY_CODE` constant/fixture instead please so it's clearer?